scripts/colmap2nerf.py

```diff
@@ -85,7 +86,12 @@ def run_colmap(args):
 	if os.path.exists(db):
 		os.remove(db)
 	do_system(f"colmap feature_extractor --ImageReader.camera_model {args.colmap_camera_model} --ImageReader.camera_params \"{args.colmap_camera_params}\" --SiftExtraction.estimate_affine_shape=true --SiftExtraction.domain_size_pooling=true --ImageReader.single_camera 1 --database_path {db} --image_path {images}")
-	do_system(f"colmap {args.colmap_matcher}_matcher --SiftMatching.guided_matching=true --database_path {db}")
+    
+	if args.colmap_matcher == "vocab_tree":
+		do_system(f"colmap {args.colmap_matcher}_matcher --VocabTreeMatching.vocab_tree_path {args.vocab_path} --SiftMatching.guided_matching=true --database_path {db}")
```

Comment:
In the interest of avoiding code duplication, could you put the command in a variable, append the vocabulary tree argument if present, and then invoke `do_system` on that variable? Thanks!